packages/tokamak/contracts-bedrock/src/universal/StandardBridge.sol

```diff
@@ -356,6 +359,27 @@ abstract contract StandardBridge is Initializable {
         // contracts may override this function in order to emit legacy events as well.
         _emitERC20BridgeInitiated(_localToken, _remoteToken, _from, _to, _amount, _extraData);
 
+        _sendERC20BridgeFinalizedMessage(_localToken, _remoteToken, _from, _to, _amount, _minGasLimit, _extraData);
+    }
+
+    /// @notice Override create finalized mesasge on other chain
+    /// @param _localToken  Address of the ERC20 on this chain.
+    /// @param _remoteToken Address of the ERC20 on the remote chain.
+    /// @param _from        Address of the sender.
+    /// @param _to          Address of the receiver.
+    /// @param _amount      Amount of the ERC20 sent.
+    /// @param _extraData   Extra data sent with the transaction.
+    function _sendERC20BridgeFinalizedMessage(
```

Comment:
I think we can delete this function and copy and paste this code in `_initiateBridgeETH`

Reply:
I updated!